homeassistant/components/sql/config_flow.py

```diff
@@ -23,6 +23,9 @@
 
 DATA_SCHEMA = vol.Schema(
     {
+        vol.Required(CONF_NAME, default="Select SQL Query"): selector.TextSelector(
+            selector.TextSelectorConfig()
```

Comment:
No need to pass this in

Reply:
I thought to do a cleanup PR after this one